packages/react-native-bridge/android/publish-aztec-and-bridge.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# 1. Verify the version argument is passed. We use the same version for react-native-aztec and react-native-bridge libraries.
+VERSION=$1
+if [[ -z $VERSION ]]; then
+    echo "This script requires the Bintray version to be passed as an argument."
+    echo "Example usage: './publish-aztec-and-bridge.sh \$VERSION'"
+    exit 1
+fi
+
+# 2. Check if we can call gradlew successfully
+./gradlew --help > /dev/null 2>&1
+gradlew_exists=$?
+if [ $gradlew_exists -ne 0 ]; then
+    echo "Gradle wrapper is not available. Make sure to run this script from '/packages/react-native-bridge/android'"
+	exit 1
```

Comment:
Looks like the indentation setting changed from spaces to tabs.

Reply:
These are hopefully fixed in https://github.com/oguzkocer/gutenberg/commit/f1df77230dbea37bb66afbe09bf713be161d8e60